### PR TITLE
LRQA-47139 | master

### PR DESCRIPTION
**Pull request by john-co**
https://issues.liferay.com/browse/LRQA-47139

### CODE REVIEW

**Comment by john-co** [comment]
test results: https://github.com/john-co/liferay-portal/pull/78

**Comment by john-co** [comment]
I'd like to backport to 7.1.x and other legacy branches as well. May have to update manually on legacy branches.

File: modules/test/poshi-runner/poshi-runner/src/test/resources/com/liferay/poshi/runner/dependencies/elements/PoshiScript.function [hunk 1]
```diff
@@ -4,7 +4,7 @@ definition {
 	function dragAndDropPortletToObject {
 		WaitForSPARefresh();
 
-		selenium.waitForText("${locator1}", "${value1}");
+		selenium.waitForTextCaseInsensitive("${locator1}", "${value1}");
```
Comment (vicnate5) [inline modules/test/poshi-runner/poshi-runner/src/test/resources/com/liferay/poshi/runner/dependencies/elements/PoshiScript.function]: @john-co Found something when trying to backport.

We cannot make changes to the poshi-runner module in master, it needs to be done through the subrepo. I suggest just not changing anything on the poshi-runner side.

Comment (john-co) [reply modules/test/poshi-runner/poshi-runner/src/test/resources/com/liferay/poshi/runner/dependencies/elements/PoshiScript.function]: removed this change from poshi-runner modules

**Comment by john-co** [comment]
@vicnate5, ready for review

**Comment by vicnate5** [comment]
Pushed to BChan.
https://github.com/brianchandotcom/liferay-portal/pull/68921
https://github.com/brianchandotcom/liferay-portal-ee/pull/23404

See 7.1.x change.

**Pull request closed without merging**